From c52987bb4c4b5289908766fcba246b11d0ffe67e Mon Sep 17 00:00:00 2001
From: Zheng Junjie <z572@z572.online>
Date: Tue, 18 Mar 2025 13:28:41 +0800
Subject: [PATCH] set -DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=OFF

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index a856369..9e15949 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -35,6 +35,7 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DAMENT_PREFIX_PATH="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=OFF  \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.47.1

